.github/workflows/checks.yml

```
@@ -1,8 +1,10 @@
-name: Mandatory Pull Request Checks
+name: Mandatory Checks
 
 on:
   pull_request:
     branches: [ "main" ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)